src/transformers/models/fast/modeling_fast.py

```diff
@@ -0,0 +1,350 @@
+        return combined_hidden_states
+
+
+class FastHead(nn.Module):
+    def __init__(self, config: FastConfig) -> None:
+        super().__init__()
+        self.conv = FastRepConvLayer(
+            config.head_conv_in_channels,
+            config.head_conv_out_channels,
+            config.head_conv_kernel_size,
+            config.head_conv_stride,
+        )
+
+        padding = get_same_padding(config.head_final_kernel_size)
+
+        self.final_conv = nn.Conv2d(
+            config.head_final_in_channels,
+            config.head_final_out_channels,
+            kernel_size=config.head_final_kernel_size,
+            stride=config.head_final_stride,
+            padding=padding,
+            bias=config.head_final_bias,
+        )
+
+        self.pooling_size = config.head_pooling_size
+        self.pooling_1s = nn.MaxPool2d(kernel_size=self.pooling_size, stride=1, padding=(self.pooling_size - 1) // 2)
+        self.pooling_2s = nn.MaxPool2d(
+            kernel_size=self.pooling_size // 2 + 1, stride=1, padding=(self.pooling_size // 2) // 2
+        )
+
+        if config.head_dropout_ratio > 0:
+            self.dropout = nn.Dropout2d(config.head_dropout_ratio)
+        else:
+            self.dropout = nn.Identity()
+
+    def forward(self, hidden_states: torch.Tensor) -> torch.Tensor:
+        hidden_states = self.conv(hidden_states)
+        hidden_states = self.dropout(hidden_states)
+        hidden_states = self.final_conv(hidden_states)
+        return hidden_states
+
+
+@add_start_docstrings(
+    """FAST (faster arbitararily-shaped text detector) proposes an accurate and efficient scene text detection
+    framework, termed FAST (i.e., faster arbitrarily-shaped text detector).FAST has two new designs. (1) They design a
+    minimalist kernel representation (only has 1-channel output) to model text with arbitrary shape, as well as a
+    GPU-parallel post-processing to efficiently assemble text lines with a negligible time overhead. (2) We search the
+    network architecture tailored for text detection, leading to more powerful features than most networks that are
+    searched for image classification.""",
+    FAST_START_DOCSTRING,
+)
+class FastForSceneTextRecognition(FastPreTrainedModel):
+    def __init__(self, config):
+        super().__init__(config)
+        self.config = config
+
+        if config.use_timm_backbone:
+            requires_backends(self, ["timm"])
+            kwargs = {}
+            if config.dilation:
+                kwargs["output_stride"] = 16
+            backbone = create_model(
+                config.backbone,
+                pretrained=config.use_pretrained_backbone,
+                features_only=True,
+                out_indices=(1, 2, 3, 4),
+                in_chans=config.num_channels,
+                **kwargs,
+            )
+        else:
+            backbone = load_backbone(config.backbone_config)
+        self.backbone = backbone
```

Comment:
Should be just like this
 
```suggestion
        self.backbone = load_backbone(config.backbone_config)
```